package.json

```diff
@@ -41,14 +41,15 @@
   },
   "build": {
     "appId": "com.symphony.electron_desktop",
-    "uninstallDisplayName": "${productName}",
     "artifactName": "${productName}-${version}-${os}.${ext}",
     "asar": true,
     "asarUnpack": "**/*.node",
     "nsis": {
```

Comment:
[Question]: we still have an nsis build entry. However, nsis target is being removed. Is there a specific reason for that?

Reply:
@sbenmoussati instead of removing `zip` I had removed nsis here.